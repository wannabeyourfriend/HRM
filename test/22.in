3
9
jump 4
copyfrom 0
outbox
inbox
copyto 0
inbox
sub 0
jumpifzero 0
jump 4