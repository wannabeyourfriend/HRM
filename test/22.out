Error on instruction 8